src/app/modules/main/chat_section/chat_content/messages/controller.nim

```diff
@@ -60,12 +60,6 @@ proc init*(self: Controller) =
       return
     self.delegate.newMessagesLoaded(args.messages, args.reactions)
 
-  self.events.on(SIGNAL_PINNED_MESSAGES_LOADED) do(e:Args):
-    let args = PinnedMessagesLoadedArgs(e)
-    if(self.chatId != args.chatId):
-      return
-    self.delegate.newPinnedMessagesLoaded(args.pinnedMessages)
```

Comment:
We no longer need this event because we don't rely on it to set the initial messages as pinned. The messages have the right property straight away.
We still need the other events about new pins and unpins though, because those can come from other users for example.

Reply:
A quick video as proof :smile: 

[pin-signals.webm](https://github.com/user-attachments/assets/c258f0f5-9732-4a6e-ba6b-1eaa483758f4)

Reply:
Nice, thank you 👍